build-docker-image.sh

```
@@ -0,0 +1 @@
+sudo docker buildx build --platform linux/amd64,linux/arm64 -t myyrakle/barus:v0.0.1 --push  .
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)